packages/remote-feature-flag-controller/src/utils/user-segmentation-utils.test.ts

```diff
@@ -7,10 +7,10 @@ import {
 } from './user-segmentation-utils';
 
 const MOCK_METRICS_IDS = [
-  '0x1234567890abcdef',
-  '0xdeadbeefdeadbeef',
-  '0xabcdef0123456789',
-  '0xfedcba9876543210',
+  '123e4567-e89b-12d3-a456-426614174000',
+  '987fcdeb-51a2-3c4b-9876-543210fedcba',
+  'a1b2c3d4-e5f6-7890-abcd-ef1234567890',
+  'f9e8d7c6-b5a4-3210-9876-543210fedcba',
```

Comment:
these are not valid UUIDv4.
- [Check 123e4567-e89b-12d3-a456-426614174000](https://generateuuid.online/validate/123e4567-e89b-12d3-a456-426614174000) is UUIDv1
- [Check 987fcdeb-51a2-3c4b-9876-543210fedcba](https://generateuuid.online/validate/987fcdeb-51a2-3c4b-9876-543210fedcba) is UUIDv3
- [Check a1b2c3d4-e5f6-7890-abcd-ef1234567890](https://generateuuid.online/validate/a1b2c3d4-e5f6-7890-abcd-ef1234567890) is UUIDv7
- [Check f9e8d7c6-b5a4-3210-9876-543210fedcba](https://generateuuid.online/validate/f9e8d7c6-b5a4-3210-9876-543210fedcba) is UUIDv3